clients/src/test/java/org/apache/kafka/clients/consumer/internals/SubscriptionStateTest.java

```diff
@@ -143,7 +153,7 @@ public void cantChangeConsumedPositionForNonAssignedPartition() {
     }
     
     public void assertAllPositions(TopicPartition tp, Long offset) {
-        assertEquals(offset, state.committed(tp));
+        assertEquals(offset, (Long) state.committed(tp).offset());
```

Comment:
Annoyingly, yes. We wouldn't need it if we changed the offset field in OffsetAndMetadata to use a Long instead of long.